pandas/tests/extension/test_numpy.py

```diff
@@ -403,6 +404,7 @@ def test_setitem_scalar_key_sequence_raise(self, data):
     def test_setitem_mask(self, data, mask, box_in_series):
         super().test_setitem_mask(data, mask, box_in_series)
 
+    # pylint: disable-next=useless-parent-delegation
```

Comment:
Same here

Reply:
Should I use `pass` instead?

Reply:
This test can be completely removed as the base class test will run